### ISSUE TITLE
Requests with KSqlDbRestApiClient can result in 431 error codes

### ISSUE THREAD

**Issue by jbkuczma** [tomasfabian/ksqlDB.RestApi.Client-DotNet#80]
I have a BackgroundService that is responsible for opening a push query and making sure it stays alive. I do this by executing `SHOW QUERIES;` to the `/ksql` endpoint and verify that the query is still running. I have noticed since this runs on a loop that after awhile I will begin to receive 431 error codes indicating that the header is too large on my request. I believe the issue is [here](https://github.com/tomasfabian/ksqlDB.RestApi.Client-DotNet/blob/25c1f83a23a204ac4887f87a1af0e5d031a73a1f/ksqlDb.RestApi.Client/KSql/RestApi/KSqlDbRestApiClient.cs#L91) and is due to the value being [appended](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.headers.httpheaders.add?view=net-8.0) to the existing header since there is already a value present.

An example output to my console when I turn on trace logging
```console
info: System.Net.Http.HttpClient.IHttpClientFactory.ClientHandler[100]
      Sending HTTP request POST /ksql

trce: System.Net.Http.HttpClient.IHttpClientFactory.ClientHandler[102]
      Request Headers:
      Accept: application/vnd.ksql.v1+json, application/vnd.ksql.v1+json, application/vnd.ksql.v1+json
      Authorization: basic [redacted]
      Content-Type: application/vnd.ksql.v1+json; charset=utf-8

```

**Comment by tomasfabian** [comment]
Hello @jbkuczma,
I released the v6.0.1 patch.

**Comment by jbkuczma** [comment]
Working for me, thank you!
